### DISCUSSION TITLE
Errors

### DISCUSSION THREAD

**Discussion by maxotru** [BlafKing/sd-civitai-browser-plus#230] [Q&A]
Hello, I can't use the extension anymore, I now have errors everywhere when I'm checking for any model, and I have the error : KeyError: 'totalPages'
What could be the problem ?
![Capture d'écran 2024-03-15 213920](https://github.com/BlafKing/sd-civitai-browser-plus/assets/157829654/f4146d32-42eb-418d-b718-b6dbf14c0178)

**Comment by BlafKing** [comment]
This issue is caused by a recent change in the CivitAI API as posted in #229

I've started working on a fix but it will take a little while since the paging system has changed which this extension relied on pretty heavily.
